atomix/README.md

```diff
@@ -0,0 +1,48 @@
+# [Atomix][Website]
```

Comment:
Any reason to keep the README untouched?

Reply:
No. I just kept it since it gives us a bit a clue what the project was and where it comes from, but I can also remove it if you insist.

Reply:
We could have a README that explains what this is - a hard fork of Atomix, why we decided to hard fork it, etc. - and link back to it. But as it is, to me this is a proper project README, so it seems strange to have that in Zeebe :sweat_smile:

Reply:
I rewrote the README